tooling/nargo_cli/build.rs

```diff
@@ -133,6 +133,8 @@ const IGNORED_NARGO_EXPAND_EXECUTION_TESTS: [&str; 8] = [
     "regression_5045",
     // bug
     "regression_7744",
+    // bug
+    "trait_associated_constant",
```

Comment:
Does this need a follow-up?

Reply:
It's this one: https://github.com/noir-lang/noir/issues/8281

I add to the ignore list instead of fixing new bugs right away because I think `nargo expand` has less priority than everything else.